lib/course.rb

```diff
@@ -1 +1,41 @@
+class Course
+  attr_reader :students
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+    @students = []
+    @full = false
+    @students_below_threshold = []
 
+    
+  end
+  def name
+    @name
+  end
+  def capacity
+    @capacity
+  end
+  def students
+    @students
+  end
+  def full?
+    @full
+  end
```

Comment:
keep an eye on your indentation:
```suggestion
    @students_below_threshold = []
  end
  
  def name
    @name
  end
  
  def capacity
    @capacity
  end
  
  def students
    @students
  end
  
  def full?
    @full
  end
```
etc.